src/condor_contrib/bosco/bosco_cluster

```diff
@@ -839,10 +841,13 @@ fi
 start_ssh
 
 # Quickly test the ssh
-qmgr_out=`ssh -o "PasswordAuthentication=no" $remote_host "pwd" 2>/dev/null`
+ssh_opts='-o "PasswordAuthentication=no"'
```

Comment:
```suggestion
ssh_opts='-o PasswordAuthentication=no'
```

Does it really work with the `""`s in there like that?

Reply:
Dunno honestly, I just grabbed it from the old line

Reply:
Well, I guess if you try it and it works i'd be curious, but i don't think it will work with the double quotes in there like that.  You have

```
ssh_opts='-o "PasswordAuthentication=no"'
qmgr_out=$(ssh $ssh_opts $remote_host "pwd" 2>&1)
```

and `$ssh_ops` will expand with the double-quotes as part of the arguments, effectively like

```
ssh $ssh_opts -o '"PasswordAuthentication=no"' $remote_host "pwd" 2>&1
```

Which, unless ssh has more smarts than i realized, will get parsed as if `"PasswordAuthentication` is the option name and  `no"` is the value, which I think ssh will give you an error for.

On the other hand, the `"`s were not necessary in the first place since `PasswordAuthentication=no` does not contain any user input or special characters that need to be quoted.